Scripts/LastArrayElement/LastArrayElement.py

```diff
@@ -0,0 +1,8 @@
+import demistomock as demisto
+
+VALUE = demisto.args()['value']
```

Comment:
same comments here

Reply:
I changed 'value' to lowercase in the source